gulp/tasks/build.js

```diff
@@ -7,7 +7,6 @@ gulp.task( 'build',
     'styles',
     'scripts',
     'images',
-    'copy',
-    'usage'
+    'copy'
```

Comment:
> This rule is needed for IE8- support.

Booooo. We should have an exception for code that doesn't run in-browser, but I don't know how we'd handle that on a practical level.